app/views/application/personal-details/residency.njk

```diff
@@ -10,13 +10,19 @@
 {% endblock %}
 
 {% block primary %}
+
+<p class="govuk-body">To get help with student visas and your immigration status, <a href=\"https://beta-getintoteaching.education.gov.uk/helping-you-become-a-teacher\">register with Get Into Teaching</a> . You will be able to speak to an adviser.</p>
+<p class="govuk-body">Most School Centred Initial Teacher Training (SCITT) providers cannot sponsor student visas for trainees on unsalaried courses.</p>
+<p class="govuk-body">You may be eligible for another type of visa that does not need your training provider to act as sponsor.</p>
+<p class="govuk-body"><a href=\"https://beta-getintoteaching.education.gov.uk/international-candidates\">Find out about student visas, visa routes that do not need a sponsor and visa options for trainees on salaried courses</a>.</p>
```

Comment:
You don’t need to escape double quotes outside of a Nunjucks component (a backwards slash is used to ‘escape’ a character that would otherwise be interpreted as part of the component’s syntax, not the content of a variable – which is probably a really bad explanation of what’s going on here) 

```suggestion
<p class="govuk-body">To get help with student visas and your immigration status, <a href="https://beta-getintoteaching.education.gov.uk/helping-you-become-a-teacher">register with Get Into Teaching</a> . You will be able to speak to an adviser.</p>
<p class="govuk-body">Most School Centred Initial Teacher Training (SCITT) providers cannot sponsor student visas for trainees on unsalaried courses.</p>
<p class="govuk-body">You may be eligible for another type of visa that does not need your training provider to act as sponsor.</p>
<p class="govuk-body"><a href="https://beta-getintoteaching.education.gov.uk/international-candidates">Find out about student visas, visa routes that do not need a sponsor and visa options for trainees on salaried courses</a>.</p>
```